### PR TITLE
docs -  gpcheckcat: add test orphaned_toast_tables

### PR DESCRIPTION
**Pull request by mkiyama**
The Note contains conditional text.

OSS text will be
If a manual change is required, update the pg_depend TOAST table entry and set the refobjid field to the correct dependent table. 

Pivotal text will be
If a manual change is required, contact Pivotal Support.

doc change for PR https://github.com/greenplum-db/gpdb/pull/6722

### CODE REVIEW

File: gpdb-doc/dita/utility_guide/admin_utilities/gpcheckcat.xml [hunk 1]
```diff
@@ -113,21 +113,38 @@
...
+            <p><codeph>duplicate</codeph> - Check for duplicate entries</p>
+            <p><codeph>foreign_key</codeph> - Check foreign keys</p>
+            <p><codeph>inconsistent</codeph> - Cross consistency check for master segment
+              inconsistency</p>
+            <p><codeph>missing_extraneous</codeph> - Cross consistency check for missing or
+              extraneous entries</p>
+            <p><codeph>owner</codeph> - Check table ownership that is inconsistent with the master
+              database</p>
+            <p><codeph>orphaned_toast_tables</codeph> - Check for orphaned TOAST tables.
+              <note>Mismatched orphaned TOAST tables due to a <codeph>reltoastrelid</codeph> entry
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/admin_utilities/gpcheckcat.xml]: Not sure what "Mismatched" means in this case.  Maybe edit for clarity:  "A TOAST table can become orphaned if the reltoastrelid entry in pg_class points to an incorrect TOAST table. These errors require a manual catalogue change."

Comment (mkiyama) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpcheckcat.xml]: clarified note text.

Comment (mkiyama) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpcheckcat.xml]: @kalensk 
Is the updated text clearer? 

The diff is kind of hard to read. Here is the updated text.

NOTE: A TOAST table can become orphaned if the reltoastrelid entry in pg_class points to an incorrect TOAST table (a TOAST table mismatch). Fixing this type of orphaned TOAST table requires a manual catalog change. If a manual change is required, 
<OSS> update the pg_depend TOAST table entry and set the refobjid field to the correct dependent table.<pivotal> contact Pivotal Support.

Comment (kalensk) [reply gpdb-doc/dita/utility_guide/admin_utilities/gpcheckcat.xml]: Looks good! Thanks!

**Pull request merged**